build/deploy.sh

```diff
@@ -10,4 +10,4 @@ fi
 
 # Build the image
 
-make IMAGE_REPOSITORY=$IMAGE_REPOSITORY build
+make IMAGE_REPOSITORY=$IMAGE_REPOSITORY build skopeo-push
```

Comment:
The `skopeo-push` target in `standard.mk` and `create_catalog_image` in `functions.mk` both are doing `skopeo copy`, I presume in the same way.  The `create_catalog_image` define is called from target `build-catalog-image` in `standard.mk`.

This is just to say we have multiple ways of doing the same thing defined.  Is there anything that will call target `build-catalog-image`?  If not then it should be removed.  If so, we have a risk of having these commands diverging.